bootstrap/mac.sh

```diff
@@ -6,3 +6,13 @@ fi
 
 brew install augeas
 brew install dialog
+
+if ! hash pip 2>/dev/null; then
+    echo "pip Not Installed\nInstalling python from Homebrew..."
+    brew install python
```

Comment:
@kuba, by `sudo easy_install virtualenv` ( uses the easy_install module default to OS X)